x-pack/test/security_solution_cypress/cypress/e2e/investigations/timelines/unsaved_timeline.cy.ts

```diff
@@ -146,8 +141,11 @@ describe.skip('Save Timeline Prompts', { tags: ['@ess'] }, () => {
 
 // In serverless it is not possible to use the navigation menu without closing the timeline
 describe('Save Timeline Prompts', { tags: ['@serverless'] }, () => {
-  before(() => {
+  beforeEach(() => {
```

Comment:
same here, do you think we should delete all created timelines before doing anything, using this [deleteTimelines](https://github.com/elastic/kibana/blob/main/x-pack/test/security_solution_cypress/cypress/tasks/api_calls/common.ts#L116) function?